### DISCUSSION TITLE
Auto Budget: Update rollover amount after first day of month

### DISCUSSION THREAD

**Discussion by Genygo** [firefly-iii/firefly-iii#5845] [Firefly III]
Hi there :)
Thanks for a great piece of software. Really helps me with my money!
Following thing i like to solve:
When I or my bank is late with the import, it happens that transaction from eg. February are added to the February budget on the second of March. 
When using the auto budget function with the roll over my march budget is already calculated. 
Is there a method to update the march budget after adding transactions to last month budget?

Best Regards :)

**Comment by JC5** [comment]
No, sorry, you’ll have to correct the budget yourself.
